src/test/javascript/pageobjects/page.ts

```diff
@@ -0,0 +1,15 @@
+import { browser } from '@wdio/globals';
+
+/**
+ * main page object containing all methods, selectors and functionality
+ * that is shared across all page objects
+ */
+export default class Page {
+  /**
+   * Opens a sub page of the page
+   * @param path path of the sub page (e.g. /path/to/page.html)
+   */
+  public open(path: string) {
+    return browser.url(`https://the-internet.herokuapp.com/${path}`);
```

Comment:
Can definitely delete this stuff. It's just example code. Figured it might be helpful, but it's not hard to create a new project if we want to see this again.

Reply:
@bprize15 @jfkonecn if you think the information is useful, please add a Test section in the root readme.